account_fiscal_year_closing/views/account_move_views.xml

```diff
@@ -9,7 +9,7 @@
         <field name="inherit_id" ref="account.view_move_form"/>
         <field name="arch" type="xml">
             <field name="ref" position="after">
-                <field name="closing_type"/>
+                <field name="closing_type" required="True"/>
```

Comment:
This is redundant since the field is already defined as required, can you remove it?

Reply:
Done, thanks!